.github/workflows/lint.yml

```diff
@@ -70,9 +70,8 @@ jobs:
           poetry run flake8 --count --show-source --statistics
 
       - name: Lint python code (mypy)
-        uses: jpetrucciani/mypy-check@0.971
-        with:
-          path: "."
+        run: |
+          poetry run mypy .
```

Comment:
Because I had seen that the latest release was version `0.971 `and I hadn't looked at the tags, can we use tags in GitHub workflows even if there hasn't been a release?

Reply:
Yes we can, thanks.